src/changes/changes.xml

```diff
@@ -134,6 +134,9 @@ The <action> type attribute can be add,update,fix,remove.
       <action issue="IO-625" dev="ggregory" type="fix" due-to="Mikko Maunu">
         Corrected misleading exception message for FileUtils.copyDirectoryToDirectory.
       </action>
+      <action issue="IO-626" dev="Yuji Konishi" type="fix" due-to="Yuji Konishi">
```

Comment:
Normally the **dev** is a committer, and **due-to** is the PR contributor name/alias.

Reply:
Sorry.

I thought that a committer and the PR contributor are me.
Which is my mistake, a committer or the PR contributor?